flask_webapp/models/log_devices.py

```diff
@@ -0,0 +1,34 @@
+import sqlite3
+import sys
+
+
+def log_devices(name, status):
```

Comment:
is this previously paired, or scanned or connected or all??? Can you make this api more descriptive via either commenting or naming convention (preferred)?